app/main.py

```diff
@@ -1 +1,45 @@
+class Animal:
+
+    def __init__(self, name: str, appetite: int, is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
```

Comment:
The `print_name` method is correctly implemented, but consider adding a return type hint for consistency with other methods.